tests/arrays_test/chunk.test.js

```diff
@@ -0,0 +1,39 @@
+const { chunk } = require('../../methods/arrays/chunk');
```

Comment:
you can import chunk method from lodash and compare the results of execution your method and lodash method